### PR TITLE
Update inventory

### PR DESCRIPTION
**Pull request by gandelman-a**

### CODE REVIEW

File: inventory/group_vars/vagrant [hunk 1]
```diff
@@ -13,8 +13,6 @@ elasticsearch_jvm_heap_size: 250m
 filebeat_output_logstash_hosts:
   - elk.vagrant:5044
 
-hoist_repo: /vagrant/.git
```
Comment (mlangbehn) [inline inventory/group_vars/vagrant:16]: This line also appears in `inventroy/group_vars/all` line 8:

```yaml
hoist_repo: https://github.com/BonnyCI/hoist.git
```

With `hoist_repo` removed from `bastion.yml`, I believe we no longer need that reference either?

File: roles/ansible-runner/tasks/runner.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    repo: "{{ item.repo }}"
+    dest: "/opt/source/{{ item.name }}"
+    update: true
+
+- name: Install config file
+  template:
+    src: ansible-runner-config
+    dest: /etc/default/{{ item.name }}
+    mode: 0600
+    owner: "{{ item.user }}"
```
Comment (mlangbehn) [inline roles/ansible-runner/tasks/runner.yml:12]: Is there a reason we are starting to work in `/opt/source/`, and then moving to `/etc/default/`, and then back to `/opt/source/`? I would suggest either moving this block up to the beginning, or after `Install playbook deps`.

File: roles/ansible-runner/templates/ansible-runner-config [hunk 3]
```diff
@@ -1,3 +1,3 @@
-ANSIBLE_PLAYBOOK={{ ansible_playbook }}
-ANSIBLE_INVENTORY={{ ansible_inventory }}
-ANSIBLE_SSH_USER={{ ansible_remote_user | default('') }}
+ANSIBLE_PLAYBOOK={{ item.playbook }}
+ANSIBLE_INVENTORY=/opt/source/{{ item.name }}/inventory/{{ item.inventory }}
+ANSIBLE_SSH_USER={{ item.user | default('')}}
```
Comment (mlangbehn) [inline roles/ansible-runner/templates/ansible-runner-config:3]: Shouldn't this be:

```jinja2
ANSIBLE_SSH_USER={{ item.user | default('') }}
```

File: roles/ansible-runner/tasks/runner.yml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    requirements: "/opt/source/{{ item.name }}/requirements.txt"
+  when: requirements_txt.stat.exists
+
+- name: Ensure cron log path
+  file:
+    state: directory
+    path: "/var/www/html/cron-logs/{{ item.name }}"
+    mode: 0755
+    owner: "{{ item.user }}"
+    group: www-data
```
Comment (mlangbehn) [inline roles/ansible-runner/tasks/runner.yml:37]: Should this go after the `include: secret-file.yml`? Just to help keep the tasks organized.

File: inventory/host_vars/bastion.vagrant [hunk 5]
```diff
@@ -1,4 +1,15 @@
...
 
-ci_inventory: vagrant
+ansible_runner_tasks:
+    - name: system-ansible
+      playbook: bastion.yml
+      inventory: vagrant
+      repo: /vagrant/.git
+      user: root
+
+    - name: cideploy-v2
```
Comment (mlangbehn) [inline inventory/host_vars/bastion.vagrant:11]: Should this be?:

```yaml
    - name: cideploy
```

**Review by mlangbehn — COMMENTED** [review]
Please see inline comments.

File: roles/ansible-runner/tasks/runner.yml (hunk 2 above)
Comment (gandelman-a) [reply roles/ansible-runner/tasks/runner.yml:12]: This was copied/pasetd from main.yml, i suppose the ordering just happened?  Will update here, tho

File: roles/ansible-runner/tasks/runner.yml (hunk 4 above)
Comment (gandelman-a) [reply roles/ansible-runner/tasks/runner.yml:37]: ditto ^

**Pull request closed without merging**